### PR TITLE
Support install/uninstall multiple addons

### PR DESCRIPTION
**Pull request by myakove**
Add support to install/uninstall multiple add-ons at once.
Support install/uninstall add-ons in parallel

### CODE REVIEW

**Review by rnetser — APPROVED** [review]

File: app/addon_cmds/commands.py [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+def run_action(action, addons, parallel, timeout):
+    jobs = []
+    for values in addons.values():
+        addon_action_func = getattr(values["cluster_addon"], action)
+        _args = [True, timeout]
+        if action == "install_addon":
+            _args.insert(0, values["parameters"])
+
+        if parallel:
+            job = Process(target=addon_action_func, args=tuple(_args))
```
Comment (oharan2) [inline app/addon_cmds/commands.py:19]: Not sure if it should be implemented here or within openshift-ci config, but do you think we should provide a flag for each process that will set to fail the whole flow if an addon installation is failing or passing?

For example while installing RHOAM&RHODS, RHODS has exceeded timeout, and I want the whole flow to continue and not teardown.

Comment (myakove) [reply app/addon_cmds/commands.py]: Update the code, if one of the jobs failed we exit

File: app/addon_cmds/commands.py [hunk 2]
```diff
@@ -6,8 +7,49 @@
 from ocm_python_wrapper.ocm_client import OCMPythonClient
 
 
+def run_action(action, addons, parallel, timeout):
+    jobs = []
+    for values in addons.values():
+        custer_addon_obj = values["cluster_addon"]
+        addon_action_func = getattr(custer_addon_obj, action)
```
Comment (omrirh) [inline app/addon_cmds/commands.py:14]: ditto

**Review by rnetser — DISMISSED** [review]

**Review by rnetser — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**